frontend/src/pages/home/aiFlows/CreateAndTrainGallery.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import { ProjectObjectType, SectionType, typedObjectImage } from '~/concepts/design/utils';
+import DividedGallery from '~/concepts/design/DividedGallery';
+import InfoGalleryItem from '~/concepts/design/InfoGalleryItem';
+import { SupportedArea } from '~/concepts/areas';
+import useIsAreaAvailable from '~/concepts/areas/useIsAreaAvailable';
+
+const CreateAndTrainGallery: React.FC<{ onClose: () => void }> = ({ onClose }) => {
+  const { status: workbenchesAvailable } = useIsAreaAvailable(SupportedArea.WORKBENCHES);
+  const { status: pipelinesAvailable } = useIsAreaAvailable(SupportedArea.DS_PIPELINES);
+
+  if (!workbenchesAvailable && !pipelinesAvailable) {
+    return null;
+  }
+
+  const workbenchesCards = workbenchesAvailable ? 1 : 0;
+  const pipelineCards = pipelinesAvailable ? 2 : 0;
```

Comment:
Can we instead of "manually counting" we create an array if `InfoGalleryItem`s and just do a `.length` on each for this calculation?

This would be effectively just shifting the render logic of the items to before the component return.